Alignment/CommonAlignment/python/tools/trackselectionRefitting.py

```diff
@@ -10,7 +10,9 @@ def getSequence(process, collection,
                 openMassWindow = False,
                 cosmicsDecoMode = False,
                 cosmicsZeroTesla = True,
-                momentumConstraint = None):
+                momentumConstraint = None,
+                cosmicTrackSplitting = False,
+                used0cut = True):
```

Comment:
please add also documentation for this parameter
in addition, could you consider renaming it because it looks at first sight like 'used 0 cut', while it means 'use d0 cut'